grails-app/controllers/org/pih/warehouse/LoggingInterceptor.groovy

```diff
@@ -31,19 +38,21 @@ class LoggingInterceptor {
             MDC.put('queryString', request?.queryString ?: "No query string")
             MDC.put('serverUrl', Holders.grailsApplication.config?.grails?.serverURL ?: "No server URL")
         } catch (Exception e) {
-            LoggingInterceptor.log.warn("Error occurred while adding attributes to Mapped Diagnostic Context: ${e.message}", e)
-
+            log.warn("Error occurred while adding attributes to Mapped Diagnostic Context: ${e.message}", e)
         }
+
+        // Now that we've populated the MDC, log the request itself, including query parameters.
+        String queryString = StringUtils.isBlank(request?.queryString) ? "" : "?${request?.queryString}"
+        log.info("${request.requestURI}${queryString} [user:${session?.user?.username}, location:${session?.warehouse?.name}]")
```

Comment:
I moved what was in `AccessLogInterceptor` to here since it felt like it should be encompassed by the logging interceptor. I also added the query parameters to the end of the request.